src/Point.ts

```diff
@@ -1 +1,38 @@
-export class Point {}
+export class Point {
+  x: number;
+  y: number;
+  point: string;
+
+  constructor();
+  constructor(x: number, y: number); 
+  constructor(x?, y?) {
+    this.x = x !== undefined ? x : 0;
+    this.y = y !== undefined ? y : 0;
+    this.point = `${this.x}, ${this.y}`;
```

Comment:
Once again, what will be in that case
```typescript
const t = new Point()
console.log(t);
t.x=10;
console.log(t);
```
What you will see in the output and why?